src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+									"List Of Available Technology Choices (As Of TripleA 1.0.3.4)");
+						break;
+					case 9: // showInfoMessage("Unit Name: The name of the unit that is included in the production frontier. Examples: infantry, artillery, and armour.\r\n\r\nTo have the program automatically add all the units to the production frontier being shown, click the 'Auto-Fill' button between the Back and Next buttons.\r\n\r\nNote: A production frontier is just a list of units that a player is allowed to purchase.", "Help For Current Step");
+						break;
+					case 10:
+						showInfoMessage(
+									"Here is a list of all the available unit attachments: movement, attack, defense, isAir, isSea, isAA, isFactory, canBlitz, isSub, canBombard, isStrategicBomber, isTwoHit, isDestroyer, isArtillery, isArtillerySupportable, isMarine, isInfantry, isParatroop, isMechanized, transportCapacity, transportCost, carrierCapacity, carrierCost",
+									"List Of Available Unit Attachment Choices (As Of TripleA 1.0.3.4)");
+						break;
+					case 11: // showInfoMessage("To change the production value of a territory, click on the territory and enter the territory's new production value in the window that appears.", "Help For Current Step");
+						break;
+					case 12: // showInfoMessage("To add a new canal, click on both of the land territories that form the canal(Touch the canal) and click yes when it asks for confirmation. To remove all the canals from the map, right click on one of the territories and click 'Yes' to confirm.", "Help For Current Step");
+						break;
+					case 13: // showInfoMessage("To change the initial owner of a territory, click on the territory and type in its new owner when it asks for it.", "Help For Current Step");
+						break;
+					case 14: // showInfoMessage("To change the units that a territory begins with, click on the territory and use the window that opens to set the territory's units. To do so, use the + and - buttons below each unit's name to change the quantity of each unit in the territory. You can also edit the text in the result textbox directly to change the territory units. Just type the name of each unit you want to add, followed by ':', followed by the unit quantitiy: Example: 'infantry: 1, artillery: 3, armour: 2, fighter: 1'.", "Help For Current Step");
+						break;
+					case 15:
+						final String[] settings = GameSettingsPanel.s_allSettings.split(", ");
+						Arrays.sort(settings);
+						StringBuilder sb = new StringBuilder();
+						sb.append("<html>Here is a list of all the available game settings:<table border=0><tr><td><table border=0>");
+						int setting_nr = 1;
+						char last_first_letter = 'z';
+						final int third_size_setting = (settings.length + 2) / 3;
+						for (final String setting : settings)
+						{
+							char first_letter = setting.charAt(0);
+							if (first_letter == last_first_letter)

```

Comment:
will do 
(comment "Build alphabetically sorted table in HTML of the setting names
in which the first letter is displayed bigger in case it is
different to the previous first letter to improve readability")